daemon/logger/awslogs/cloudwatchlogs.go

```diff
@@ -350,6 +355,7 @@ func ValidateLogOpt(cfg map[string]string) error {
 		case logGroupKey:
 		case logStreamKey:
 		case regionKey:
+		case "tag":
```

Comment:
@samuelkarp that's a good point… not sure why this (and `labels`/`env`) are not constant. But this could be done in a follow-up PR :angel: